stock_vlm_mgmt/models/stock_location_vlm_tray_type.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2019 Camptocamp SA
+# Copyright 2024 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class StockLocationVlmTrayType(models.Model):
+    _name = "stock.location.vlm.tray.type"
+    _description = "VLM Tray configuration"
+    _rec_names_search = ["name", "code"]
+
+    active = fields.Boolean(default=True)
+    name = fields.Char(required=True)
+    code = fields.Char(required=True)
+    rows = fields.Integer(required=True)
+    cols = fields.Integer(required=True)
+    width = fields.Integer(help="Width of the tray in mm")
+    depth = fields.Integer(help="Depth of the tray in mm")
+    height = fields.Integer(help="Height of the tray in mm")
+    width_per_cell = fields.Float(compute="_compute_width_per_cell")
+    depth_per_cell = fields.Float(compute="_compute_depth_per_cell")
+    tray_matrix = fields.Serialized(compute="_compute_tray_matrix")
+
+    @api.depends("width", "cols")
+    def _compute_width_per_cell(self):
+        for record in self:
+            width = record.width
+            if not width:
+                record.width_per_cell = 0.0
+                continue
+            record.width_per_cell = width / record.cols
```

Comment:
Please prevent division by zero.
```suggestion
            record.width_per_cell = record.cols and (width / record.cols) or 0
```
<img width="1067" height="170" alt="image" src="https://github.com/user-attachments/assets/dd47155d-0a8d-4d40-8a02-745473381cd1" />

Reply:
I added the condition to the `if` above, IMO it looks cleaner